### ISSUE TITLE
[T3] [Accessibility] [Functional] Unable to navigate to refresh button using tab

### ISSUE THREAD

**Issue by naveennayakp** [Mohanme13/Feb-19-Bug-bash#165]
Unable to navigate to refresh button using tab
